src/workerd/server/workerd-api.c++

```diff
@@ -944,24 +949,27 @@ kj::Own<jsg::modules::ModuleRegistry> WorkerdApi::initializeBundleModuleRegistry
         break;
       }
       case config::Worker::Module::COMMON_JS_MODULE: {
-        // TODO(soon): These are intentionally commented out for the time
-        // being and will be soon handled in a follow up PR. This branch
-        // is not yet taken in production.
-        // bundleBuilder.addSyntheticModule(
-        //     def.getName(), jsg::modules::Module::newCjsStyleModuleHandler<
-        //         jsg::CommonJsModuleContext,
-        //         JsgWorkerdIsolate_TypeWrapper>(
-        //             kj::str(def.getCommonJsModule()),
-        //             kj::str(def.getName())));
+        kj::Vector<kj::String> exports(def.getNamedExports().size());
+        for (auto name: def.getNamedExports()) {
+          exports.add(kj::str(name));
```

Comment:
Vector has a length when you define it, and add here just appends it. Shouldn't this be exports[i]?

Reply:
No, with `kj::Vector` that size argument on the constructor is a reserve. It does not allocate those slots... specifically, if I do:

```
kj::Vector<int> vec(1);
vec[0] = 2;
```

Then it will throw an exception for index out of bounds.